src/containerapp/azext_containerapp/commands.py

```diff
@@ -222,4 +222,9 @@ def load_command_table(self, args):
         g.custom_show_command('list', 'list_session_pool')
         g.custom_command('create', 'create_session_pool', supports_no_wait=True)
         g.custom_command('update', 'update_session_pool', supports_no_wait=True)
-        g.custom_command('delete', 'delete_session_pool', confirmation=True, supports_no_wait=True)
\ No newline at end of file
+        g.custom_command('delete', 'delete_session_pool', confirmation=True, supports_no_wait=True)
+
+    with self.command_group('containerapp java logger', is_preview=True) as g:
+        g.custom_command('set', 'create_or_update_java_logger', is_preview=True)
```

Comment:
The `set` command support no wait?

Reply:
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/6ef463f1-4b97-4e04-8f6c-41a6125dae6c)
Looks it support, please add `supports_no_wait=True`